src/main/io/vtx_smartaudio.c

```diff
@@ -71,9 +72,9 @@ static const char * const saPowerNames[] = {
 static const vtxVTable_t saVTable;    // Forward
 static vtxDevice_t vtxSmartAudio = {
     .vTable = &saVTable,
-    .capability.bandCount = 5,
```

Comment:
i've not sure using vtx_band_count here is a good idea since the amount of channels and bands is device dependant.  on firmware that supports optional rtc6705 it doesn't make sense it limit both vtx devices to the same amount of bands and channels.

what if one vtx provider added a band or channel that the other couldn't support?  how would VTX_BAND_COUNT be set at compile time?

Reply:
In "io/vtx_settings_config.h" the #ifdefs could be set to different values using the VTX_SMARTAUDIO / VTX_TRAMP / VTX_RTC6705 flags, the way VTX_SETTINGS_POWER_COUNT is set now.  It's tricky because the same build needs to support both SmartAudio and Tramp vid-transmitters, as either one can be connected at run time.  Is there a different way you'd like to see these defined values be setup?

Reply:
i think the original way, where the driver itself knows how many channels and bands are supported is correct, if the smartaudio system needs to know about how many are supported internally then it can either reference the vtxSmartAudio structure or use a #define that ONLY used by the smartaudio system.  there should be no cross dependencies between the different vtx systems, the vtxDevice is the abstraction.

Reply:
Hello I tried to plug in and arm and set one of stew'sat UAFeatures  99$ builds  up graded model F3 Flight Controller Acro 6 DOF/Deluxe 10 DOF for Multirotor RacingEachine VTX03 Super Mini 5.8G 72CH 0/25mW/50mw/200mW Switchable FPV TransmitterEachine VTX03 Super Mini 5.8G 72CH 0/25mW/50mw/200mW Switchable FPV Transmitter PDB-XT60 W/ BEC 5V & 12V 2oz Copper stack in it want to set switch arm flysky AFNDS-2A Transmmiter and receaver  itconnected but no set up screen.